src/include/mpir_request.h

```diff
@@ -222,10 +222,11 @@ static inline int MPIR_Request_is_persistent(MPIR_Request * req_ptr)
  * A handle is "active" if it is neither null nor "inactive". */
 static inline int MPIR_Request_is_active(MPIR_Request * req_ptr)
 {
-    if (req_ptr == NULL)
+    if (req_ptr == NULL) {
```

Comment:
If this is going to be a new requirement, you're going to want to figure out how to add checking for it to the pre-commit hook (which should feed the style checker script in Jenkins). Otherwise this will be out of date really fast.

Reply:
Yes, I would like to have that requirement. While it is not difficult to let `add_func_log.pl` to add braces when necessary, it is difficult to ensure `remove_func_log.pl` to return the previous code.

I think I can come up with a perl script checking that you can put in the pre-commit hook. I'll add it to this PR. Once we use the script, we can easily add additional style checking that indent fails to do.

Reply:
I don't mind adding that requirement. As long as we set them to "cuddle", then it doesn't make much difference in terms of line counts (except where people put things on the same line as the `if`/`else`.

I was hoping that GNU Indent would add the braces if you used `-br`, but it looks like that will only move the braces around if they already exist, not add new ones. Rolling our own version is likely to be error prone as we'll find new ways to break it. It also just means more time in the pre-commit hook checking everything since there'll be two scripts to run now instead of one.

Reply:
FYI, I've thrown this out to the Internet for help finding the answer in case there is one: https://stackoverflow.com/questions/59757949/have-gnu-indent-add-new-braces-after-if-statement

Reply:
Oh, by the way, when you run `add_func_log.pl`, it will warn you when it encounters those `if (...) return ...` cases, so at least it won't "silently" break your code :)